src/mapper/dynakubes.go

```diff
@@ -56,6 +56,7 @@ func (dm DynakubeMapper) UnmapFromDynaKube() error {
 	}
 	for _, ns := range nsList {
 		delete(ns.Labels, dtwebhook.InjectionInstanceLabel)
+		ns := ns
 		setUpdatedViaDynakubeAnnotation(&ns)
```

Comment:
I think using the oldschool way with the index is much more intuitive in this situation
```suggestion
		setUpdatedViaDynakubeAnnotation(&nsList[i])
```

Reply:
Matter of taste, I prefer `ns := ns` because if explicitly and directly addresses memory aliasing problem. Also syntax is just easier.